python/ray/dashboard/dashboard.py

```diff
@@ -27,13 +26,24 @@
 import grpc
 from google.protobuf.json_format import MessageToDict
 import ray
+
 from ray.core.generated import node_manager_pb2
 from ray.core.generated import node_manager_pb2_grpc
 from ray.core.generated import reporter_pb2
 from ray.core.generated import reporter_pb2_grpc
 from ray.core.generated import core_worker_pb2
 from ray.core.generated import core_worker_pb2_grpc
-import ray.ray_constants as ray_constants
+from ray.dashboard.interface.dashboard_controller_interface import (
+    BaseDashboardController)
+from ray.dashboard.interface.dashboard_route_handler_interface import (
+    BaseDashboardRouteHandler)
+try:
+    from ray.dashboard.metrics_exporter.exporter import Exporter
+    from ray.dashboard.metrics_exporter.client import MetricsExportClient
+except ImportError:
+    print("Pydantic is not donwloaded. pip install pydantic")
```

Comment:
Given that pydantic is used for (static) type checking, can we strip the type information from the metrics exporter? It would be good if it works without that dependency. Maybe we can have a script that just strips it for the wheels, and in the development version the types are there or something like that.

Reply:
I might misunderstand, but are you talking about those parts that have `variable: Type`? I thought pydantic will be used just for request/response schema, and regular type checking will be still there because it helps us to maintain code better. Is having this type annotation decreasing performance or something?

Reply:
We actually cannot install pydantic here because it's 3.6+

Reply:
Will use the hand-written class that have APIs that we are using currently.